src/v3/src/components/Widget/index.tsx

```diff
@@ -188,33 +201,29 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
         setIdxTransaction(await triggerEmailVerifyCallback(widgetProps));
         return;
       }
-      const transaction: IdxTransaction = await authClient.idx.start({
+      transaction = await authClient.idx.start({
         stateHandle,
         // Required to prevent auth-js from clearing sessionStorage and breaking interaction code flow
         exchangeCodeForTokens: false,
       });
-      const hasError = !transaction.requestDidSucceed || transaction.messages?.some(
+      const hasError = !transaction!.requestDidSucceed || transaction!.messages?.some(
```

Comment:
Are the `!` non-null assertions needed? I expect the return type of `await authClient.idx.start` would be `idxTransaction` with no possibility of `null | undefined`